site_admin_token.py

```diff
@@ -37,13 +37,15 @@ def get_site_admin_token(username=None, password=None, refresh_token=None):
             )
 
         if "error" in credentials:
+            os.remove("tmp/site-admin-refresh-token")
             return get_site_admin_token()
 
         with open(f"tmp/site-admin-refresh-token", "w") as f:
             f.write(credentials["refresh_token"])
 
         return credentials["access_token"]
     except Exception as e:
+        os.remove("tmp/site-admin-refresh-token")
```

Comment:
If this crashes (e.g. because the file does not exist), it'll crash out the call to `get_site_admin_token()`. Recommend just placing a try block around this so it continues even if it fails.